### PR TITLE
Adds missing versions in py-libsonata

### PR DESCRIPTION
**Pull request by sergiorg-hpc**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-libsonata/package.py [hunk 1]
```diff
@@ -12,6 +12,13 @@ class PyLibsonata(PythonPackage):
     homepage = "https://github.com/BlueBrain/libsonata"
     git = "https://github.com/BlueBrain/libsonata.git"
 
+    version('develop', branch='master', submodules=False, get_full_repo=True)
```
Comment (sergiorg-hpc) [inline var/spack/repos/builtin/packages/py-libsonata/package.py:15]: The `develop` version was also missing for some reason!

File: var/spack/repos/builtin/packages/py-libsonata/package.py [hunk 2]
```diff
@@ -12,6 +12,13 @@ class PyLibsonata(PythonPackage):
     homepage = "https://github.com/BlueBrain/libsonata"
     git = "https://github.com/BlueBrain/libsonata.git"
 
+    version('develop', branch='master', submodules=True, get_full_repo=True)
```
Comment (sergiorg-hpc) [inline var/spack/repos/builtin/packages/py-libsonata/package.py:15]: The `develop` version was also missing for some reason!

**Comment by matz-e** [comment]
Oh Jenkins, retest this please

**Comment by sergiorg-hpc** [comment]
@matz-e Something went wrong with the CI yesterday, but I guess it was a random thing. Should we still merge or relaunch again?

**Review by matz-e — APPROVED** [review]
Fine to merge, that one test is not required.

**Pull request merged**